.github/workflows/test_codecov.yml

```diff
@@ -0,0 +1,41 @@
+name: CI test and coverage
+
+on:
+  pull_request
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7, 3.8, 3.9]
+    name: Run tests for ${{ matrix.python-version }}
+    steps:
+    - uses: actions/checkout@v2
+    - name: Install python
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+        architecture: x64
+    - name: Install dependencies and package
+      run: |
+        python -m pip install --upgrade pip
+        pip install -r requirements.txt
+        pip install -r requirements-dev.txt
+        pip install -e .
+    - name: Test with pytest
+      run: pytest
+    - name: Flake8
+      run: flake8
+    - name: Test coverage
+      run: |
+        py.test --cov=era5cli --cov-report term --cov-report xml:cov.sml tests/
```

Comment:
Doesn't this work if you #dropthedot? https://stackoverflow.com/a/41893170

Reply:
This was copied verbatim from the Travis file. Dropping it!